### DISCUSSION TITLE
Can’t load repository: Couldn't connect to server

### DISCUSSION THREAD

**Discussion by BeesUser** [codemagic-ci-cd/codemagic-docs#2505] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Branch with /
example
my/main/888_fix

### Expected results

Success cloning and build

### Actual results

Cloning into '/Users/builder/clone'...
fatal: unable to access 'git.url' Failed to connect to git.url port 443 after 75088 ms: Couldn't connect to server

### Build id (optional)

66349b20175c08465e52f60a

**Comment by himesh-cm** [comment]
Hey @KropVOo, I see that you were able to resolve the issue. Please feel free to reach out if you are still looking for help.

**Reply by BeesUser** [reply]
the problem occurs periodically
build id = 663cad89287db72c79c09fd1

**Reply by BeesUser** [reply]
and again 
663ea027791960843ff9a19f

**Comment by mikhail-tokarev** [comment]
Hi @KropVOo 

The problem not related to the branch name, but rather your firewall settings for your repository.
Some time ago we've added new data centers and you need to allow their IP addresses through the firewall (whitelist).

More information you can find here: https://docs.codemagic.io/getting-started/adding-apps/#firewall-configuration-for-privately-hosted-repositories

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
